### PR TITLE
Add some language about json-patches

### PR DESCRIPTION
**Pull request by OR13**
Related:
- https://github.com/decentralized-identity/sidetree/issues/359
- https://github.com/decentralized-identity/sidetree/issues/449

### CODE REVIEW

File: docs/did-state-patches.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+                    },
+                ],
+            }
+    ]
+}
+```
+
+Sidetree DID Methods MAY choose to implement support for `json-patches` actions as did state patches. `patches` must contain [rfc6902](https://tools.ietf.org/html/rfc6902) compliant json patches.
+
+Sidetree Method Implementers MUST ensure that all possible combinations of supported DID State Patches, and intermediary states, that are reachable do not introduce security vulnerabilities, or terminal states from which no new operations further operations are valid.
```
Comment (bstasyszyn) [inline docs/did-state-patches.md]: Typo: 'no new operations further operations'

File: docs/did-state-patches.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+Method Implementers are always responsible for ensuring that protocol rules are enforced. However, it is up method implementers to decide what the desired behavior is with respect with linear appliation of state transitions.
```
Comment (bstasyszyn) [inline docs/did-state-patches.md]: Typo: it is up to method ...

Comment (bstasyszyn) [inline docs/did-state-patches.md]: Typo: with respect to linear application ...

File: scripts/build-spec.js [hunk 3]
```diff
@@ -1,4 +1,6 @@
+const { exec } = require("child_process");
 require('spec-up')();
+
 setTimeout(()=>{
-    process.exit(0);
+    exec('killall node')
```
Comment (OR13) [inline scripts/build-spec.js]: very aggressive.

**Comment by csuwildcat** [comment]
I already added a json patch section, it's in master

**Comment by OR13** [comment]
I will close this once  I open another PR to update the patches section of the spec... I think the current language needs some adjustment.

**Pull request closed without merging**